ReactApp/src/components/Editor.jsx

```diff
@@ -132,13 +128,12 @@ function Editor({ post }) {
 
 	const settings = useMemo(
 		() => ({
-			...editorSettings,
```

Comment:
@geriux as described in the pull request description, I believe this is [redundant](https://github.com/WordPress/gutenberg/blob/219065c3be5881a68a8667fda9c04c7b7f546763/packages/editor/src/components/provider/index.js#L178). Even worse, it creates infinite render loops when `useSubRegistry` is `false`. Will you please help me verify I am not introducing regressions with this change?

Reply:
Thank you for addressing this! I've confirmed it does not introduce any regressions 🚀